component-runtime-beam/src/main/build/ClassesIndices.groovy

```diff
@@ -92,13 +92,18 @@ def excludedPackages = [
     'org.mortbay.',
     'org.kohsuke.',
     // todo: check
+    'org.apache.avro.hadoop.',
+    'org.apache.avro.ipc.',
+    'org.apache.avro.mapred.',
+    'org.apache.avro.mapreduce.',
     'org.apache.zookeeper.',
     'org.apache.hadoop.',
     'org.apache.directory.',
     'com.codahale.metrics.'
 ]
 
 def simplifiedPackages = [
+    'org.apache.beam.repackaged', // we want everything inside repackaged
```

Comment:
Likely the wrong fix, guess https://github.com/pbailly/component-runtime/blob/ee4d7479fa79341d457fb11b22ad8694b99b8781/component-runtime-beam/src/main/build/ClassesIndices.groovy#L155 should let classes pass with this end dot so likely one of the previous substring messes up thing
Will have a look later this week since all that code will need to be adjusted for 1.1.13 anyway